editor/editor_inspector.cpp

```diff
@@ -3864,14 +3864,16 @@ void EditorInspector::set_use_filter(bool p_use) {
 }
 
 void EditorInspector::register_text_enter(Node *p_line_edit) {
-	search_box = Object::cast_to<LineEdit>(p_line_edit);
+	search_box = Object::cast_to<DebouncedLineEdit>(p_line_edit);
 	if (search_box) {
-		search_box->connect(SceneStringName(text_changed), callable_mp(this, &EditorInspector::_filter_changed));
+		search_box->connect("debounce_timeout", callable_mp(this, &EditorInspector::update_tree));
+		return;
 	}
-}
 
-void EditorInspector::_filter_changed(const String &p_text) {
-	update_tree();
+	search_box = Object::cast_to<LineEdit>(p_line_edit);
```

Comment:
I don't get why this method takes Node, but I kept support for the old behavior just in case.